src/preact.d.ts

```diff
@@ -11,7 +11,8 @@ declare namespace preact {
 	interface PreactHTMLAttributes {
 		dangerouslySetInnerHTML?:DangerouslySetInnerHTML;
 		key?:string;
-		ref?:(el?: Element) => void;
+		defaultValue?:string;
```

Comment:
Could someone answer why its called DomAttributed rather than DomAttributes? Its kind of confusing on the first look.